db/models.py

```diff
@@ -19,8 +19,8 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(Actor, related_name="genre")
-    genres = models.ManyToManyField(Genre, related_name="actor")
+    actors = models.ManyToManyField(Actor, related_name="genres")
+    genres = models.ManyToManyField(Genre, related_name="actors")
```

Comment:
Please, read my comment and checklist again
You were supposed to change it to `movies`. You will use it like this: `actor_object.movies.all()` and get all movies that are related to actors